src/init.cpp

```diff
@@ -1104,6 +1107,44 @@ bool AppInitLockDataDirectory()
     return true;
 }
 
+/**
+ * Once initial block sync is finished and no change in block height or queued downloads,
+ * sync utxo state to protect against data loss
+ */
+static void SyncCoinsTipAfterChainSync(const NodeContext& node)
+{
+    LOCK(node.chainman->GetMutex());
+    if (node.chainman->IsInitialBlockDownload()) {
+        LogDebug(BCLog::COINDB, "Node is still in IBD, rescheduling chainstate disk sync...\n");
```

Comment:
Can we clarify this message a bit to emphasize that this rescheduling does not preclude periodic flushes to disk?

It's slightly confusing to see this message as it appears that disk flushes are being deferred indefinitely.

See: https://github.com/bitcoinknots/bitcoin/issues/70#issuecomment-2136078696

Reply:
Updated the log message to be `"Node is still in IBD, rescheduling post-IBD chainstate disk sync..."`. Does that clarify it?